scripts/kconfig/kconfigfunctions.py

```diff
@@ -335,6 +335,15 @@ def dt_compat_enabled(kconf, _, compat):
     return "n"
 
 
+def cmake_in_list(kconf, _, elem, list):
```

Comment:
Could call it `in_cmake_list()` maybe. I read it as `cmake_in_list` as "CMake is in the list" at first and was a bit confused.

Reply:
I though about it, but finally opted to use cmake as namespace to follow logic used for other dt functions (`dt_foo`, ..).
Up to you!

Reply:
How about `cmake_list_contains`?